receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/constant.go

```diff
@@ -23,6 +23,49 @@ const (
 	AttributeECSTaskRevesion = "ecs.task-definition-version"
 	AttributeECSServiceName  = "ecs.service"
 
-	ContainerMetricsLabelLen = 3
-	TaskMetricsLabelLen      = 6
+	CPUsInVCpu = 1024
+	BytesInMiB = 1024 * 1024
+
+	TaskPrefix                        = "ecs.task."
+	ContainerPrefix                   = "container."
```

Comment:
Why do we need to prefix the metrics with container? If they have container label, they're container metrics right?

Reply:
I'm not sure if this part of the OTel convention but given that other receivers follow this approach, I think we should do the same here for consistency.

Reply:
Yea, I found some other receivers are doing the same like `kubeletstatsreceiver` and `dockerstatsreceiver`.